lib/alma/bib_item.rb

```diff
@@ -120,6 +120,10 @@ def missing_or_lost?
       !!process_type.match(/MISSING|LOST_LOAN/)
     end
 
+    def techserv_unassigned_or_intref?
+      !!holding_location.match(/techserv|UNASSIGNED|intref/)
```

Comment:
In this case you should use `.match?` that will return false for 0 matches instead of `!!` which is cryptic.
```suggestion
      holding_location.match?(/techserv|UNASSIGNED|intref/)
```